src/styles/modules/divider.scss

```diff
@@ -0,0 +1,41 @@
+.feature-products-text p {
+   position: relative;
+   &::after {
+   content: '';
+   display: block;
+   width: 132px; 
+   height: 3px; 
+   background-color:#ff2d2d;
+   position: absolute;
+   left: 44%;
+   margin-top: 29px;
+  }
+}
+
+.feature-products {
+   position: absolute;
+   &::before {
+   content: '';
+   display: block;
+   height: 2px; 
+   background-color: #ececec; 
+   margin-top: 108px;
+   position: relative;
+   }
+}
+
+.meet-team p {
+   position: relative;
+   &::after {
```

Comment:
you have the same styles for `.feature-products-text p` above, it would be more reusable to make a separate class with this styles and apply it anywhere you need, not only bind it to `p`